Source/WebCore/page/SecurityOriginData.h

```diff
@@ -42,6 +49,14 @@ struct SecurityOriginData {
     {
         RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(!isHashTableDeletedValue());
     }
+    SecurityOriginData(MarkableOpaqueOriginIdentifier opaqueOriginIdentifier)
+        : protocol(emptyString())
+        , host(emptyString())
+        , port(std::nullopt)
+        , opaqueOriginIdentifier(opaqueOriginIdentifier)
+    {
+        RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(!isHashTableDeletedValue());
```

Comment:
Why this new RELEASE assertion? Can we omit or turn it into a debug assertion?

Reply:
It shouldn't be needed, given that we're explicitly initializing the value. I'll delete it.